site/profile/hiera.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
What part of the hiera hierarchy is touching `site/profile/hiera.yaml`?

Reply:
This is module level hiera